indico/modules/rb_new/client/js/components/RoomBasicDetails.jsx

```diff
@@ -80,11 +93,32 @@ export default class RoomBasicDetails extends React.PureComponent {
         return (
             <Grid columns={2}>
                 <Grid.Column textAlign="center" styleName="photo-column">
-                    <div styleName="image-container">
-                        <SpriteImage pos={room.spritePosition}
+                    <Modal open={openImageModal}
+                           onClose={() => this.setState({openImageModal: false})}>
+                        <Modal.Content image styleName="image-dimmer">
+                            <Dimmer.Dimmable>
+                                <Dimmer active={!imageLoaded} inverted>
+                                    <Loader />
+                                </Dimmer>
+                                <Image wrapped
+                                       onLoad={() => this.setState({imageLoaded: true})}
+                                       src={fetchRoomImageURL({room_id: room.id})} />
+                            </Dimmer.Dimmable>
+                        </Modal.Content>
+                    </Modal>
+                    <div styleName={isClickable ? 'image-container clickable-image' : 'image-container'}>
```

Comment:
```jsx
<div styleName=`image-container {isClickable ? 'clickable-image' : ''}`>
```

Reply:
Is that valid JSX?

Reply:
Also, does this work with the styleName magic which gets resolved at compile time?

Reply:
Forgot the `$` and brackets.
I'd swear we have this structure in other places

Reply:
For instance - https://github.com/indico/indico/blob/master/indico/modules/rb_new/client/js/common/timeline/TimelineLegend.jsx#L33

Reply:
Not sure if css module name mangling is applied to the conditional class name...

Reply:
why not use `className={toClasses{...`?

Reply:
className/toClasses don't support css modules

Reply:
I believe that what @mvidalgarcia suggested would work. But it's a matter of trying it out 😉

Reply:
I followed another route in the end that didn't require me to try that. (Having a different render method and not returning the Modal in cases of unclickable like it's  been mentioned in other comments)